solutions/c/01-oo8/diff/app/main.c.diff

```diff
@@ -0,0 +1,18 @@
+@@ -1,15 +1,11 @@
+ #include <stdio.h>
+
+ int main() {
+-  // You can use print statements as follows for debugging, they'll be visible when running tests
+-  printf("Logs from your program will appear here!\n");
+-
+-  // Uncomment this block to pass the first stage
+-  //   printf("$ ");
+-  //   fflush(stdout);
++    printf("$ ");
```

Comment:
@glokta1 again, not self-reviewing thoroughly enough - see the indentation mismatch here? This is one of the first things a user will see when they use CodeCrafters, and it'll look very amateur to mess up indentation